content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Netzgrafik-Editor"
+linkTitle: "Netzgrafik-Editor"
+weight: 40
+description: "Open-source software developped by SBB CFF FFS and its integration in OSRD"
+---
+
+Netzgrafik-Editor (NGE) is an open-source software that enables the creation, modification, and analysis of regular-interval timetable, at a macroscopic level of details, developped by [Swiss Federal Railways (SBB CFF FFS)](https://www.sbb.ch/). See [front-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend) and [back-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend) repositories.
+
+OSRD (microscopic level of detail, trains scheduled once, based on a defined infrastructure, depicts a timetable) and NGE (macroscopic level of detail, regular-interval based train runs, no infrastructure, depicts a transportation plan) are semantically different, but close enough to make it work together.
+The compatibility between NGE and OSRD has been tested through by a prove of concept, by running both softwares as separate services and not automated synchronization.
+
+The idea is to give to OSRD a graphical tool to edit (create, update and delete train schedules from) a timetable from a operational study scenario, and get some insights on analytics at the same time. The second benefit of using both microscopic and macroscopic level of detail is that OSRD microscopic calculations can be propagated in NGE to enhance the level of detail of it.
+
+The transversal objective of this feature is to make 2 open-source projects from 2 big infrastructure managers work along and cooperate for one another with the same goals.
+
+#### 1 - Integration in OSRD
+
+Therefore, NGE is integrated in the operational studies section of OSRD, in an [`iframe`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe). This `iframe` points to `osrd-nge`, a minimalist Angular app that embed the actual NGE app. It can also be seen as a wrapper of the actual NGE app. `osrd-nge` then customizes the actual NGE app with specific parameters and functionnalities:
```

Comment:
Have we decided sth on an `<iframe>` vs. a new tab? That said, either way, wouldn't change anything substantial.